src/zif_abapgit_definitions.intf.abap

```diff
@@ -346,6 +346,25 @@ INTERFACE zif_abapgit_definitions
     tty_lines TYPE STANDARD TABLE OF i
                         WITH NON-UNIQUE DEFAULT KEY .
 
+  TYPES:
```

Comment:
can these definitions be moved to a new interface under src/apack/ ? this object is already too big, and it looks like these new type definitions are independent

Reply:
Sure, no problem. I just thought, you wanted to have everything centralized. I will do the changes once we have slightly more clarify about the direction which we are taking...

Reply:
to me, this is the only thing blocking this PR from being merged. 

I dont like the concept with the interface, but hey, we can try it out anyhow

Reply:
Of course - changes will be done. However, currently having some issues synchronizing the changes on my local instance with my fork on GitHub and the upstream changes. I'll let you know once everything is done...